### ISSUE TITLE
explain base-aepp not being secure

### ISSUE THREAD

**Issue by thepiwo** [aeternity/aepp-base#443]

**Comment by j28** [comment]
@thepiwo could you provide more detail on what kind of explanation you are referring to (ex. browser not as as secure as app... or not as secure as ledger/airgap vault... or something else)?

**Comment by thepiwo** [comment]
@j28 yes the idea was to explain that the base-aepp is meant as a daily spending wallet and should not hold a lot of tokens in any case, also as it can't be as secure as a vault, due to it living in the browser and on an internet connected device.

There were a few thoughts about only showing this once there are X tokens stored, but we didn't agree on something here, we wanted to let this decision to someone specifying or designing the task better.

**Comment by j28** [comment]
thank you. this is clear. if i have any more questions, i'll reach out again.
